examples/using-page-transitions/src/components/transition.js

```diff
@@ -3,9 +3,24 @@ import {
   TransitionGroup,
   Transition as ReactTransition,
 } from "react-transition-group"
-import getTransitionStyle from "../utils/getTransitionStyle"
 
 const timeout = 250
+const getTransitionStyles = () => {
```

Comment:
nitpicky - this doesn't have to be function that creates new object each time `render` is called if output will always be the same - this can be just plain object